test/dtypes/test_affine_quantized_float.py

```diff
@@ -25,10 +25,11 @@
 from torch._inductor.test_case import TestCase as InductorTestCase
 from torch.testing._internal import common_utils
 
-from torchao.dtypes.floatx.float8_layout import Float8AQTTensorImpl, preprocess_scale
+from torchao.dtypes.floatx.float8_layout import preprocess_scale
```

Comment:
it's a bit confusing on why this file changes at all, if we are keeping BC for now with the old logic.  I would have expected this file to stay as-is to provide coverage for the old path, and new tests to be added to cover the new path.  WDYT?

Reply:
we can split the deprecation to a separate PR, but I was planning to just deprecate the old path since the exposure is still limited at this point and it's good that we don't maintain multiple code paths for the same thing